core/vidl/vidl_ffmpeg_istream_v0.9.txx

```diff
@@ -474,7 +474,7 @@ vidl_ffmpeg_istream::current_frame()
 
     // If the pixel format is not recognized by vidl then convert the data into RGB_24
     vidl_pixel_format fmt = vidl_pixel_format_from_ffmpeg(enc->pix_fmt);
-    if (fmt == VIDL_PIXEL_FORMAT_UNKNOWN)
+    if (fmt == VIDL_PIXEL_FORMAT_UNKNOWN || fmt == VIDL_PIXEL_FORMAT_YUV_420P)
```

Comment:
This was an old change that I kept to myself.  With the little recollection that I have, I believe I ran into a some problem when decoding a raw video stream that was encoded in YUV_420P format.  After adding this or condition, I was able to get the full RGB image decoded.   I had to admit that the code in this way reads a little odd though.